### PR TITLE
added with-react-foundation example

### PR DESCRIPTION
**Pull request by therealmarzouq**
## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have helpful link attached, see `contributing.md`

## Documentation / Examples

- [ ] Make sure the linting passes by running `yarn lint`

### CODE REVIEW

File: examples/with-react-foundation/.gitignore [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# local env files
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```
Comment (balazsorban44) [inline examples/with-react-foundation/.gitignore:31]:
```suggestion
.env.*local
```

File: examples/with-react-foundation/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# React-foundation Example
```
Comment (balazsorban44) [inline examples/with-react-foundation/README.md:1]:
```suggestion
# react-foundation Example
```

File: examples/with-react-foundation/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "foundation-sites": "6.6.3",
+    "react-foundation": "^0.9.7"
+  },
+  "devDependencies": {
+    "@types/node": "^14.6.0",
+    "@types/react": "^17.0.3",
+    "@types/react-dom": "^17.0.3",
+    "typescript": "4.3.2"
```
Comment (balazsorban44) [inline examples/with-react-foundation/package.json:19]: Could we make sure these are on their latest versions as of writing?

File: examples/with-react-foundation/src/pages/index.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import Link from 'next/link'
+import { Article } from '../components/Article'
+import { Header } from '../components/Header'
+import { Callout } from 'react-foundation'
+
+// Add Foundation to index.js
+import 'foundation-sites/dist/css/foundation.min.css'
```
Comment (balazsorban44) [inline examples/with-react-foundation/src/pages/index.tsx:7]: Why not in `_app`? I assume other pages that want to use `react-foundation` would need this as well.

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Thanks, added some small comments.

File: examples/with-react-foundation/src/pages/index.tsx (hunk 4 above)
Comment (therealmarzouq) [reply examples/with-react-foundation/src/pages/index.tsx]: Thanks, I have done this

File: examples/with-react-foundation/package.json (hunk 3 above)
Comment (therealmarzouq) [reply examples/with-react-foundation/package.json]: I have made this update

**Comment by therealmarzouq** [comment]
> Thanks, added some small comments.

Hello, They've been resolved

**Review by balazsorban44 — APPROVED** [review]
Thanks!

**Pull request merged**